webpack.config.js

```diff
@@ -18,14 +18,6 @@ module.exports = {
   module: {
     loaders: [
       {
-        test: /\.js$/,
-        // exclude: /(node_modules)/,
-        loader: 'babel',
```

Comment:
Oh, I made a comment above about how we _should_ be using babel in the webpack build, not realizing that we already did. We definitely should not be deleting this, otherwise `solid-client` won't work in browsers.